src/transbank.c

```diff
@@ -107,6 +107,57 @@ BaseResponse *parse_load_keys_close_response(char *buf)
   return response;
 }
 
+TotalsResponse *parse_get_totals_response(char *buf)
+{
+  TotalsResponse *response = malloc(sizeof(TotalsResponse));
+
+  char *w;
+  int i = 1, l = 0, f = 0;
```

Comment:
Took me awhile to figure what was doing each of this variables, it would be a good idea to use more descriptive names, to make our life easier in the future.